Link to Interactive Teleport Labs

benarent opened the pull request:
This PR makes it a little easier to find our interactive learning tracks. This is the first version, I've plan to improve the landing page but this should provide a nice getting started for Teleport Application Access and Teleport Server Access users.

benarent committed:
Add title and note for Instruqt

ptgott reviewed (commented):
Should we also add a link to the "Getting started" section of `/docs/application-access/introduction`? Same with the `introduction` pages for Kubernetes Access and Server Access.

benarent committed:
Added Title to Teleport Application Access and fix 'gif'

benarent commented:
I've added a link to Teleport Application Access page, but will create a different PR for Teleport Kubernetes Access as I'll need to get an icon for it. 

I also noticed the 'gifs' were broken, so I fixed that as part of this PR.

dumez-k reviewed (approved):
🚢

benarent committed:
Merge branch 'master' into benarent/instrqut
